k-distribution/src/main/scripts/bin/krun

```diff
@@ -70,13 +70,15 @@ execute () {
 
 KX=$(basename "$0")
 
+fold_lines="fold -s"
+
 error () {
-  printf "[Error] Critical: $@\n" | fold -s 1>&2
+  printf "[Error] Critical: $@\n" | $fold_lines 1>&2
   exit $result
 }
 
 warning () {
-  printf "[Warning] Compiler: $@\n" | fold -s 1>&2
+  printf "[Warning] Compiler: $@\n" $fold_lines 1>&2
```

Comment:
```suggestion
  printf "[Warning] Compiler: $@\n" | $fold_lines 1>&2
```

I think you deleted the `| ` which you need?

Reply:
Right, I missed that.